.github/workflows/debian9.yml

```diff
@@ -47,4 +47,3 @@ jobs:
         run: ./ci/test_integration_espnet1.sh
       - name: test espnet2 integration
         run: ./ci/test_integration_espnet2.sh
```

Comment:
Do we need to use `codecov-action` for debian and centos test?

Reply:
Well, I'm still thinking about that. We might get more coverage if there are difference (but no). On the other hand, there are potential risks to collapse coverage if we accidentally have inconsistent codecov usage (e.g. flags, coverage erase) in the additional files. So I decided to remove codecov from other yml.